src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificatePal.Windows.cs

```diff
@@ -544,5 +551,25 @@ public byte[] Export(X509ContentType contentType, SafePasswordHandle password)
                 return exported;
             }
         }
+
+        private unsafe T AcquireCertContext<T>(CertContextCallback<T> callback)
```

Comment:
Nit: The Aquire naming makes me think there should be a corresponding Release.  Seems like it'd be better named something like "InvokeWithCertContext", since the acquire/release to get the context happens around the invocation of the callback.